### DISCUSSION TITLE
Ecowitt WH51 europe version frequency

### DISCUSSION THREAD

**Discussion by CiccioDroid** [merbanan/rtl_433#2202] [Q&A] [answered]
Hello everybody,
I have bought an ecowitt wh51 european version with 868mhz frequency, but if i set 868.3M 868.8M and 868.0M with sample rate of 250k i'm not receive any messages.
Do You have any suggestions?

**Comment by zuckschwerdt** [comment, marked as answer]
Try to find the signal first, see e.g. https://triq.org/rtl_433/ANALYZE.html#verify-a-transmission

**Reply by CiccioDroid** [reply]
Found 867.3M

**Reply by zuckschwerdt** [reply]
That seems rather far off, but your RTL-SDR might not report an accurate frequency.
Tune a little to the side, e.g. if the signal is at exactly `867.3M` then use `867.35M` or `867.4M`

**Comment by gdt** [comment]
For what it's worth, my experience with Ecowitt transmitters is on 915 MHz (US).   Actually measuring the frequency is difficult.   However, I used 3 quality RTL-SDR dongles (SmarTEE v2, Smart v5, Blog v3) to look at a number of Ecowitt, Acurite, and TPMS transmitters.   My conclusion was
  - the 3 quality dongles were within 1 ppm or so of each other
  - the Ecowitt stuff was very accurate in frequency, as was Ford TPMS
  - the acurite stuff was close enough and with good dongles set to 433.92 worked fine
  - two lesser dongles were off, by roughly 60 and 90 ppm.  This is a vast amount at 868 or 915 Mhz

So @CiccioDroid  cannot conclude that the transmitter is off until measuring a whole bunch of other stuff with the same dongle, and trying to calibrate.   Doing this right is tricky, and even though I have radio experience, it took me several hours of thought.

So yes, by all means tune so you receive the signal.  But look at the freq1/freq2 values, and make sure it's mid-band not at the edge.    I find that freq1 is below tune and freq2 above for Ecowitt and that's been reliable.

If you have  dongle that is not frequency stable, I recommend getting a Blog v3 or a Smart v4/v5.
